### PR TITLE
added rate-us/contact-us form

### PR DESCRIPTION
**Pull request by Aditijainnn**
image
image
also in console inputs are coming 
#11

### CODE REVIEW

**Comment by AlfiyaSiddique** [comment]
@Aditijainnn This is not a contact form just the rating form,  so make the button feature text and form head text  as `Rate Us`
Make Name and rating mandatory without those values the form should not get submit

**Comment by Aditijainnn** [comment]
> @Aditijainnn This is not a contact form just the rating form, so make the button feature text and form head text as `Rate Us` Make Name and rating mandatory without those values the form should not get submit

hey i have made all the changes please review and merge it

**Comment by Aditijainnn** [comment]
@AlfiyaSiddique please accept my pr

File: frontend/src/Components/Footer.jsx [hunk 1]
```diff
@@ -9,47 +8,177 @@ import {
...
+    setRating(rate);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault(); // Prevent form from refreshing the page
+
+
+     // Check for mandatory fields
+     if (!name || !email || !message || rating === 0) {
+      alert("Name, Email, Message, and Rating are mandatory fields!");
```
Comment (AlfiyaSiddique) [inline frontend/src/Components/Footer.jsx:29]: use toast instead of alert

**Comment by AlfiyaSiddique** [comment]
@Aditijainnn one comment please fix it

File: frontend/src/Components/Footer.jsx (hunk 1 above)
Comment (Aditijainnn) [reply frontend/src/Components/Footer.jsx]: done

**Review by AlfiyaSiddique — APPROVED** [review]

**Pull request merged**